docs/source/install/installing_mala.rst

```diff
@@ -7,11 +7,16 @@ Prerequisites
 MALA supports any Python version starting from ``3.10.4``. No upper limit on
 Python versions are enforced. The most recent *tested* version is ``3.10.12``.
 
-MALA requires ``torch`` in order to function. As the installation of torch
-depends highly on the architecture you are using, ``torch`` will not
-automatically be installed alongside MALA. Please obtain a suitable version
-of ``torch`` from the `official website <https://pytorch.org/>`_ before
-continuing.
+MALA requires ``torch`` (https://pytorch.org). Most install sites will want to
+use an already installed ``torch`` or install a specific version before
+installing the MALA package itself, for instance
+
+.. code-block:: sh
+
+    pip install --index-url https://download.pytorch.org/whl/cu<cuda_version> torch==<torch_version>
+
+If no ``torch`` is found, we install the latest GPU-enabled version via ``pip
+install torch`` (see ``requirements.txt``).
```

Comment:
Doesn't that read like the user has to do that manually, whereas with this PR this will be done be done automatically?

Reply:
The intent was encourage people to install a version of torch that matches their hardware, but well people who are aware of the torch install procedure do that already, and those who aren't are probably fine if we `pip install torch` for them. I'll rephrase the section.

Reply:
Thank you!